src/components/Ranking/Ranking.js

```diff
@@ -0,0 +1,80 @@
+import * as classes from './style.module.scss';
+
+export const RANKING_COMPONENT_NAME = 'ranking-component';
+
+class Ranking extends HTMLElement {
+  constructor() {
+    super();
+
+    this._contents = new DocumentFragment();
+
+    this.container = Ranking.addContainer();
+
+    this.titleElement = Ranking.addTitle();
+    this.container.appendChild(this.titleElement);
+
+    this.rankingList = Ranking.createRankingList(this.playerScores);
+    this.container.appendChild(this.rankingList);
+
+    this._contents.appendChild(this.container);
+  }
+
+  connectedCallback() {
+    this.appendChild(this._contents);
+  }
+
+  setUserScores(playerScores) {
+    this.playerScores = playerScores;
```

Comment:
Did you test that? I think that after updating values you also need to rebuild (render) whole element